### ISSUE TITLE
usb fat device could not be opened on recent versions

### ISSUE THREAD

**Issue by Tasiobg** [FIX94/Nintendont#368]
Hi
Today after updating Nitendont to the last version I can see this error : "usb fat device could not be opened", I have reinstalled v4.421 and is working again. I'm using an external hardrive with a 32Gb Fat32 partition.

Hope that help with the development and thanks very much for this masterpiece.

**Comment by rosshadden** [comment]
@Tasiobg Is `421` the latest that worked, or did you just go back far enough that you wouldn't have to worry about it?

**Comment by Tasiobg** [comment]
@rosshadden I think 421 is the first version I have ever installed so I went back to this one, there may be other newer versions workin as well.

**Comment by rosshadden** [comment]
Okay thanks. I installed 421 last night and it did indeed get passed that error... And into a memory dump 🤣 when trying to load the USB drive. 

After upgrading to the latest nintendont again, I figured out though playing around that if I hit up or something on the dpad, the "disk info" option in the top right becomes enabled. Selecting it results in the same mem dump. 

I'll take a picture of the dump when I get back home.
